lib/index.js

```diff
@@ -1,3 +1,30 @@
 const React = require('react');
 const ReactDOM = require('react-dom');
 
+
+class HarvestHelper extends React.Component {
+  constructor() {
+    super();
+    this.state = {};
```

Comment:
Are you eventually going to save things on state? Does it need to be set no matter what? I'm wondering because I have no idea.